client/my-sites/stats/all-time/index.jsx

```diff
@@ -78,3 +97,27 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state ) => {
+	const siteId = getSelectedSiteId( state );
+	const query = {};
+	const allTimeData = getSiteStatsForQuery( state, siteId, 'stats', query ) || { stats: {} };
+	const {
+		posts,
+		views,
+		visitors,
+		views_best_day,
+		views_best_day_total
+	} = allTimeData.stats;
+
+	return {
+		requesting: isRequestingSiteStatsForQuery( state, siteId, 'stats', query ),
+		query,
+		siteId,
+		posts,
+		views,
+		visitors,
+		views_best_day,
+		views_best_day_total
```

Comment:
This looks a bit weird to me for a couple of reasons:

- Lots of props
- Naming convention of `views_best_day` is contrary to Calypso practices

Having lots of props is probably OK - this way each propType can be validated separately, but I think it would be good to fix the names here:

```js
return {
    ...
    visitors,
    viewsBestDay: views_best_day,
    viewsBestDayTotal: views_best_day_total
};
```

Alternatively, this cleanup could be done at a higher level, then you could use the spread operator to avoid some repetition.

```js
return {
   requesting: ...,
   query,
   siteId,
   ...allTimeData.stats
};
```